src/components/Sidebar/components/NavItem/NavItem.spec.js

```diff
@@ -23,16 +33,15 @@ describe('NavItem', () => {
       expect(actual).toMatchSnapshot();
     });
 
-    it('should render children', () => {
+    it('should render children when selected', () => {
       const wrapper = shallow(
```

Comment:
We don't use `shallow` anymore, could you change to `mount`?

Reply:
And this will be applied to all the tests in this file and maybe in all the navigation